Makefile

```diff
@@ -11,7 +11,7 @@ help:
 .PHONY: minikube-start
 minikube-start: # @HELP Start a local k8s cluster using minikube
 minikube-start:
-	minikube --profile minikube-wbaas start --kubernetes-version=1.26.5
+	minikube --profile minikube-wbaas start --kubernetes-version=1.26.14
```

Comment:
comment: Makes sense to bump but could perhaps could come in a separate PR. In any case we actually have split 1.26 and 1.27 in our GKE clusters right now